gstools/field/rng.py

```diff
@@ -121,30 +138,30 @@ def _create_normal_dists(self, mode_no=None):
         return Z
 
     def _prepare_random_numbers(self, mode_no):
-        """ Create and partly fill some standard arrays.
+        """Create and partly fill some standard arrays.
 
         Parameters
         ----------
-            mode_no : int, optional
+            mode_no : :class:`int`, optional
                 number of the Fourier modes
 
         Returns
         -------
-            k : ndarray
+            k : :class:`numpy.ndarray`
                 the empty mode array
-            iid : ndarray
+            iid : :class:`numpy.ndarray`
                 uniformly distributed variables
         """
         k = self.create_empty_k(mode_no)
         iid = np.empty_like(k)
 
-        for d in range(self.dim):
+        for dim_i in range(self.dim):
             rng = self._get_random_stream()
-            iid[d] = rng.uniform(0., 1., mode_no)
+            iid[dim_i] = rng.uniform(0., 1., mode_no)
```

Comment:
I think a `d` as a index variable in a short loop is perfectly fine

Reply:
Ok. That's just my handwriting